changelogs/unreleased/gh-9105-increase-error-serialization-verbosity.md

```diff
@@ -0,0 +1,5 @@
+## feature/lua
```

Comment:
Regarding the commit message.

> field followed by a tab and the rest of the fields

It seems, a tab character was there in the old version of the patch and now it is a space.